### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by v-saivamsia**

### CODE REVIEW

**Comment by saivamsi1998** [comment]
Can you update the read me

**Comment by v-saivamsia** [comment]
Yes sure

**Comment by saivamsi1998** [comment]
Ok

**Comment by saivamsi1998** [comment]
Nice work

**Comment by v-saivamsia** [comment]
Thanks! Shall I add new features

**Comment by saivamsi1998** [comment]
Yes! Please do.

**Comment by saivamsi1998** [comment]
Great

**Review by saivamsi1998 — CHANGES_REQUESTED** [review]
Please be more specific about the new feature.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# sudoku
+Hi Welcome to the sudoku solver application.
+Updating the read me.
+Adding new features.
```
Comment (saivamsi1998) [inline README.md:4]: add feature details

File: README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# sudoku
+Hi Welcome to the sudoku solver application.
+Updating the read me.
```
Comment (saivamsi1998) [inline README.md:3]: this is good

Comment (v-saivamsia) [reply README.md:3]: glad to hear

**Pull request merged**